packages/now-cli/src/commands/logs.js

```diff
@@ -283,16 +283,19 @@ function printLogShort(log) {
       ` ${obj.status} ${obj.bodyBytesSent}`;
   } else if (log.type === 'event') {
     data = `EVENT ${log.event} ${JSON.stringify(log.payload)}`;
+  } else if (obj) {
```

Comment:
I think this `obj` thing is deprecated.

Reply:
Ya I think so too. Maybe relevant for Now 1.0 deployments?